mirai-core-api/src/commonMain/kotlin/data/FriendInfo.kt

```diff
@@ -32,4 +34,5 @@ public open class FriendInfoImpl(
     override val uin: Long,
     override var nick: String,
     override var remark: String,
+    override val friendGroupId: Int,
```

Comment:
https://github.com/mamoe/mirai/blob/fc9dab80cea715bc8a627e899f4f86484dfd42f3/mirai-core-api/src/commonMain/kotlin/data/FriendInfo.kt#L25-L38

`Kept for binary compatibility.`. @Him188